Corona-Warn-App/src/main/res/layout/fragment_information_legal.xml

```diff
@@ -99,7 +99,7 @@
                     android:layout_marginTop="@dimen/spacing_tiny"
                     android:text="@string/information_legal_subtitle_contact"
                     android:focusable="true"
-                    android:autoLink="all"
+                    android:autoLink="email"
```

Comment:
`all` is equivalent to ` web|email|phone|map`
if you keep email only do the other still work?

Reply:
This TextView only displays `@string/information_legal_subtitle_contact` which only shows an email address. That's why I think it's fine to go tih `email` only. Do you agree?

Reply:
For this one I agree, I have only email address